Sources/TuistKit/Generator/GeneratorModelLoader.swift

```diff
@@ -6,17 +6,22 @@ import TuistGenerator
 
 enum GeneratorModelLoaderError: Error, Equatable, FatalError {
     case featureNotYetSupported(String)
+    case missingFile(AbsolutePath)
```

Comment:
This was added - 

Previously this class re-used `GraphLoaderError` to throw this kind of error :/